architecture/service_catalog/aws_broker.adoc

```diff
@@ -0,0 +1,73 @@
+[[admin-guide-get-started-aws]]
+= {product-title} Amazon Web Services Broker
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+The _AWS Service Broker_ provides access to Amazon Web Services (AWS) through the {product-title} Service Catalog. 
+
+[NOTE]
+====
+The AWS Service Broker is supported and qualified on OpenShift Container Platform.  It is offered directly from Amazon as a download and as such, many components of this service broker solution are supported directly by Amazon. Red Hat provides support for the installation of the AWS Service Broker and troubleshooting of the OpenShift cluster and Service Catalog issues.
+====
+
+This guide describes how to deploy the AWS Service Broker into your {product-title} cluster.
```

Comment:
s/This guide describes how to deploy/You can deploy

Reply:
@kalexand-rh Are we moving away from _This guide_ structure? It is fairly common in the docs.

Reply:
@mburke5678, I'm 99% sure that it's disallowed by the IBM style guide.